addons/pandora/model/type.gd

```diff
@@ -73,8 +73,12 @@ static func lookup(name: String) -> PandoraPropertyType:
 
 	if ResourceLoader.exists(type_path):
 		var ScriptType = load(type_path)
-		if ScriptType != null and ScriptType.has_source_code():
-			return ScriptType.new()
+		if ScriptType != null:
+			if Engine.is_editor_hint() and ScriptType.has_source_code():
```

Comment:
Why only when editor hint? Pandora always needs to run, regardless of editor or not.

Reply:
I suggest this fix instead:
```
if ScriptType != null:
   if ScriptType.has_source_code() or ScriptType.can_instantiate():
      return ScriptType.new()
```

Reply:
If it needs to run in editor the type handlers should be marked as `@tool` and just using `can_instantiate` will work.

Reply:
> If it needs to run in editor

I recommend not using `@tool` unless Godot _really_ needs to access a script at Editor time.

Reply:
That's exactly what I said or meant at least. You will have to know if you _really_ need access to those type handlers at edtior time. But from what I have seen of the addon they seem to be used to serialize values in the editor. Or am I mistaken here?

Reply:
@HolonProduction good point - I'd expect in that case not using `@tool` should produce some kind of error. If that's true, we should definitely use `@tool`

Reply:
Yeah not quite sure what is intended here either. But `can_instantiate` checks whether the script is a tool when in the editor. I guess instantiating a script which returns `false` could be called undefined behavior. But this should probably raise an error. But I don't have the time to make an MRP and report it at the moment.